.eslintrc.js

```diff
@@ -67,5 +71,8 @@ module.exports = {
     'brace-style': 'off',
     'new-cap': 'off',
     'no-continue': 'off',
+  },
+  "globals": {
+    "__DEBUG__": true
```

Comment:
Should this be `false` to disallow setting `__DEBUG__`? (so it's only set by Webpack's DefinePlugin)

(the eslint-config-airbnb-base config enables `no-global-assign` at error level to get us covered, but IIUC `true` would bypass it)